utils/docker-ce/Makefile

```diff
@@ -1,15 +1,15 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=docker-ce
-PKG_VERSION:=19.03.3
+PKG_VERSION:=19.03.4
 PKG_RELEASE:=2
 PKG_LICENSE:=Apache-2.0
 PKG_LICENSE_FILES:=components/cli/LICENSE components/engine/LICENSE
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://codeload.github.com/docker/docker-ce/tar.gz/v$(PKG_VERSION)?
-PKG_HASH:=63b0d28608f32573b9c03fa46247c6f959e9c08133ddf30a71276919de0194c0
-PKG_SOURCE_VERSION:=a872fc2f86
+PKG_HASH:=b279493f74f0d91253c7cb4b49e0b4be139de0a1d164303ce6814028d642d656
+PKG_SOURCE_VERSION:=9013bf583a
```

Comment:
It's okay we leave out `PKG_SOURCE_VERSION` since the download protocol is not git.

Reply:
This is also used in the binaries themselves. 
I figured it would be fine to overload this variable as the process wouldn't change if the download protocol was ever changed to git.

Reply:
I added a comment to help explain the other reason for its inclusion.